frontend/src/index.ts

```diff
@@ -930,7 +956,9 @@ export class App extends LiteElement {
         if (data.name === "auth_storage") {
           if (data.value !== AuthService.storage.getItem()) {
             if (data.value) {
-              this.authService.saveLogin(JSON.parse(data.value) as Auth);
+              this.authService.saveLogin(
+                JSON.parse(data.value) as unknown as Auth,
```

Comment:
Doesn't need to be passed through unknown
```suggestion
                JSON.parse(data.value) as Auth,
```